src/components/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from "react";
+import { ContainerWithCards } from "components/ContainerWithCards";
+import { Row } from "antd";
+import { Loader } from "../Loader";
+import { ErrorToast } from "components/ErrorToast";
+import { useSelector, useDispatch } from "react-redux";
+import {
+  actions,
+  popularCategoriesSelectors,
+} from "store/popularCategoriesSlice";
+import { AppDispatch } from "store/store";
+
+export const MainPage: React.FC = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const popularCategories = useSelector(
+    popularCategoriesSelectors.getPopularCategories
+  );
+
+  const loading = useSelector(popularCategoriesSelectors.getIsLoadingSelector);
+  const loaded = useSelector(popularCategoriesSelectors.getIsLoadedSelector);
```

Comment:
импорты коллекций называем с большой буквы
```suggestion
  const loaded = useSelector(PopularCategoriesSelectors.getIsLoadedSelector);
```